packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,47 @@
+import { DividerRole, ValuesOf } from '@microsoft/fast-foundation';
+import { Orientation } from '@microsoft/fast-web-utilities';
+
+/**
+ * Fast Foundation DividerRole property
+ * @public
+ */
+export { DividerRole };
+
+/**
+ * Fast Web Utilties Orientation property
+ * @public
+ */
+export { Orientation as DividerOrientation };
+
+/**
+ * Align content within divider
+ * @public
+ */
+export const DividerAlignContent = {
+  center: 'center',
+  start: 'start',
+  end: 'end',
+} as const;
+
+/**
+ * The types for DividerAlignContent
+ * @public
+ */
+export type DividerAlignContent = ValuesOf<typeof DividerAlignContent>;
+
+/**
+ * DividerAppearance - divider color defined by a design token alias.
+ * @public
+ */
+export const DividerAppearance = {
+  strong: 'strong',
+  brand: 'brand',
+  subtle: 'subtle',
+  default: 'default',
```

Comment:
@miroslavstastny Is the intent of `"default"` here essentially `undefined`? Is there a pattern for when we take an approach like `"default"` instead of `undefined`

Reply:
For all the "enums" we enumerate all the valid values, one of them it the default one which is used if no value is passed. Unfortunately, in this case the `default` is one of the valid explicit options (which is implicitly selected if no value is passed) 🤷‍♂️.
So I suggest to keep it.

Reply:
Ok